src/safeposix/syscalls/net_calls.rs

```diff
@@ -1867,6 +1901,7 @@ impl Cage {
         exceptfds: Option<&mut interface::FdSet>,
```

Comment:
Can you explain a bit about `FdSet` in the form of comments? Currently, not able to understand it without looking at its implementation.